.gitignore

```diff
@@ -12,3 +12,5 @@ test-output/
 website/node_modules
 website/build
 website/i18n/en.json
+.metals
```

Comment:
Nitpick: IMO all user specific tools (not related to CI), should be placed in personal `~/.git/ignore` file.

Reply:
Fair enough, will delete